services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -142,62 +132,106 @@ export class NoticeOfWorkTable extends Component {
   });
 
   columns = () => [
-    {
-      title: "Region",
-      dataIndex: "mineRegion",
-      render: (text) => <div title="Region">{text}</div>,
-      filteredValue: this.props.searchParams.mine_region,
-      filters: this.props.mineRegionOptions
-        ? optionsFilterAdapter(this.props.mineRegionOptions)
-        : [],
-    },
     {
       title: "NoW No.",
-      dataIndex: "nowNum",
+      key: "now_number",
+      dataIndex: "now_number",
       sortField: "now_number",
+      sorter: true,
+      ...this.filterProperties("NoW No.", "now_number"),
       render: (text, record) => (
         <Link to={this.createLinkTo(router.VIEW_NOTICE_OF_WORK_APPLICATION, record)}>{text}</Link>
       ),
-      sorter: true,
-      ...this.filterProperties("NoW No.", "now_number"),
     },
     {
       title: "Mine",
-      dataIndex: "mineName",
+      key: "mine_name",
+      dataIndex: "mine_name",
+      sortField: "mine_name",
+      sorter: true,
+      ...this.filterProperties("Mine", "mine_name"),
       render: (text, record) =>
         record.mineGuid ? (
           <Link to={router.MINE_NOW_APPLICATIONS.dynamicRoute(record.mineGuid)}>{text}</Link>
         ) : (
           <div title="Mine">{text}</div>
         ),
     },
+    {
+      title: "Region",
+      key: "mine_region",
+      dataIndex: "mine_region",
+      sortField: "mine_region",
+      sorter: true,
+      filtered: true,
+      filteredValue: this.props.searchParams.mine_region
+        ? [this.props.searchParams.mine_region]
+        : [],
+      filters: this.props.mineRegionOptions
+        ? optionsFilterAdapter(this.props.mineRegionOptions).sort((a, b) =>
+            a.value > b.value ? 1 : -1
+          )
+        : [],
+      render: (text) => <div title="Region">{text}</div>,
+    },
     {
       title: "NoW Type",
-      dataIndex: "nowType",
+      key: "notice_of_work_type_description",
+      dataIndex: "notice_of_work_type_description",
       sortField: "notice_of_work_type_description",
+      sorter: true,
+      filtered: true,
+      filteredValue: this.props.searchParams.notice_of_work_type_description
+        ? [this.props.searchParams.notice_of_work_type_description]
+        : [],
+      filters: [
+        { text: "Quarry - Construction Aggregate", value: "Quarry - Construction Aggregate" },
+        { text: "Coal", value: "Coal" },
+        { text: "Placer Operations", value: "Placer Operations" },
+        { text: "Mineral", value: "Mineral" },
+        { text: "Sand & Gravel", value: "Sand & Gravel" },
+        { text: "Quarry - Industrial Mineral", value: "Quarry - Industrial Mineral" },
```

Comment:
Please ignore the fact that these are hard-coded in for now. Should be set up to send over the codes instead of the description.

Reply:
Yeah I think there is a selector that can be used to create this exact list 👍

Reply:
I used those selectors initially but they create a map of desc:code. How the backend is set up right now it needs desc:desc.